### PR TITLE
Fix app plan sku in arm templates

### PR DESCRIPTION
**Pull request by bishal-pdMSFT**

### CODE REVIEW

File: dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json [hunk 1]
```diff
@@ -78,8 +78,8 @@
         {
             "type": "Microsoft.Web/serverfarms",
             "sku": {
-                "Tier": "[parameters('sku')]",
-                "Name": "[concat(substring(parameters('sku'), 0, 1), '1')]"
+                "Tier": "[first(take(split(parameters('sku'), ' '), 1))]",
```
Comment (sachinma) [inline dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json]: I am not sure if the logic is correct here. shouldnt' it be last(split,..,)

File: dotnet/aspnet/webapp/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (sachinma) [inline dotnet/aspnet/webapp/.gitattributes]: why is this file required?

File: dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json (hunk 1 above)
Comment (bishal-pdMSFT) [reply dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json]: i tried with this and it had worked. Let me double check the docs.

Comment (bishal-pdMSFT) [reply dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json]: last(split()) will not work in case there are more than once spaces - not seen currently

Comment (bishal-pdMSFT) [reply dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json]: my intention is to split and take 2nd element in array

File: dotnet/aspnet/webapp/.gitattributes (hunk 2 above)
Comment (bishal-pdMSFT) [reply dotnet/aspnet/webapp/.gitattributes]: this was not intended. will remove.

File: dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json (hunk 1 above)
Comment (bishal-pdMSFT) [reply dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json]: changed skip to take

Comment (bishal-pdMSFT) [reply dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json]: sorry, changed take to skip
